### DISCUSSION TITLE
How to turn on line wrap in diff

### DISCUSSION THREAD

**Discussion by Andy-AO** [extrawurst/gitui#999] [Q&A] [answered]
git diff has automatic line wrap in Windows Terminal, but not in gitui.

Does gitui diff have an line wrap feature? If so, how do I turn it on?

If not, how do I read long lines ?

I didn't find anything on the search engine, so I came here for help.

![git diff](https://user-images.githubusercontent.com/38101170/142757328-e336f780-8297-422a-87e7-20a21b751cb3.png)
![gitui](https://user-images.githubusercontent.com/38101170/142757331-0cd55188-2fdf-41e2-9626-e9c981b75b46.png)

Line wrap is also supported in lazygit.

![lazygit](https://user-images.githubusercontent.com/38101170/142757845-b6691dc0-0930-4bfe-bf9c-2f6ebf45beb5.png)

**Comment by extrawurst** [comment, marked as answer]
it cannot be configured yet. its a known limitation I want to remove once I rewrite the diff generation anyway which will allow us to do this aswell. this will happen as part of #358

**Comment by davidngo239** [comment]
It is an limitation, how to open it

**Comment by sowdowdow** [comment]
Is this limitation still present ?

**Reply by extrawurst** [reply]
yes
